homeassistant/components/gardena_bluetooth/number.py

```diff
@@ -62,11 +62,11 @@ class GardenaBluetoothNumberEntityDescription(NumberEntityDescription):
     GardenaBluetoothNumberEntityDescription(
         key=DeviceConfiguration.rain_pause.uuid,
         translation_key="rain_pause",
-        native_unit_of_measurement=UnitOfTime.DAYS,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
```

Comment:
The native unit is days and it should use that.

If you want Home Assistant to use a different unit by default, set the `suggested_unit_of_measurement` property.

../Frenck

Reply:
No, the native unit is minutes. It was wrongly assumed to be days.

Reply:
Aah check, misunderstood 👍 

../Frenck